docs/user/guides/automation-rules.rst

```diff
@@ -0,0 +1,84 @@
+How to manage versions automatically
+====================================
+
+In this guide,
+we show you how to define rules to automate creation of new versions on Read the Docs,
+using your Git repository's version logic.
+Automating your versioning on Read the Docs means you only have to handle your versioning logic in Git.
+
+.. seealso::
+
+   :doc:`/versions`
+     Learn more about versioning of documentation in general.
+
+   :doc:`/automation-rules`
+     Reference for all different rules and actions possible with automation.
+
+Adding a new rule
+-----------------
+
+#. Navigate to :menuselection:`Admin --> Automation Rules`.
+#. Click on :guilabel:`Add Rule` and you will see the following form.
+
+.. image:: /img/screenshot_automation_rules_add.png
+   :alt: Screenshot of the "Add Rule" form
+
+In the :guilabel:`Automation Rule` form, you need to fill in 4 fields:
+
+#. Enter a :guilabel:`Description` that you can refer to later.
+   For example, entering "Create new stable version" is a good title,
+   as it explains the intention of the rule.
+   You might have different rules for unstable versions (pre-releases).
+
+#. Choose a :guilabel:`Match`,
+   which is the pattern you wish to detect in either a Git branch or tag.
+
+   * :ref:`Any version <automation-rules:Predefined matches>` matches all values.
+   * :ref:`SemVer versions <automation-rules:Predefined matches>` matches only values that have the `SemVer`_ format.
+   * :ref:`Custom match <automation-rules:User defined matches>` matches your own pattern (entered below).
+     If you choose this option,
+     a field :guilabel:`Custom match` will automatically appear below the drop-down where you can add a regular expression in `Python regex format`_.
+
+#. Choose a :guilabel:`Version type`.
+   You can choose between *Tag* or *Branch*,
+   denoting *Git tag* or *Git branch*.
+
+#. Finally, choose the :guilabel:`Action`:
+
+   * :ref:`Activate version <automation-rules:Actions for versions>`
+   * :ref:`Hide version <automation-rules:Actions for versions>`
+   * :ref:`Set version as default <automation-rules:Actions for versions>`
+   * :ref:`Delete version (on branch/tag deletion) <automation-rules:Actions for versions>`
+
+
+Now your rule is ready and you can press :guilabel:`Save`.
+The rule takes effect immediately when a new version is discovered.
+
+.. tip::
+
+   Examples of common usage
+     See :ref:`the list of examples <automation-rules:Examples>` for rules that are commonly used.
+
+   Want to test if your rule works?
+     You can test if your rule works in a few minutes without disturbing your readers:
+     If you are using Git tags in order to create new versions,
+     create a Git tag that matches the rule and check if your automation action is triggered.
+     After the experiment,
+     you can delete the Read the Docs version and the Git tag.
```

Comment:
I find this tip to be useful, however I don't know if the process is the best way of testing a rule. Are there other ways of checking if a rule triggers?